scripts/package-helm

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+set -e
+
+if ! hash helm 2>/dev/null; then
+    exit 0
+fi
+
+cd $(dirname $0)/..
+. ./scripts/version
+
+rm -rf build/charts
+mkdir -p build dist/artifacts
+cp -rf charts build/
+
+sed -i \
+    -e 's/version:.*/version: '${HELM_VERSION}'/' \
+    -e 's/appVersion:.*/appVersion: '${HELM_VERSION}'/' \
```

Comment:
The current chart has these as different values https://github.com/rancher/system-charts/blob/b7b2d22a13944536dc2889beda4a7855456340db/charts/rancher-gke-operator/1.0.100/Chart.yaml#L7-L8

Is the current chart wrong, or is this wrong?

Reply:
Talked about this offline.